commands/project/dev/unifiedFlow.ts

```diff
@@ -0,0 +1,169 @@
+import path from 'path';
+import util from 'util';
+import { ArgumentsCamelCase } from 'yargs';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import { isTranslationError } from '@hubspot/project-parsing-lib/src/lib/errors';
+import { translateForLocalDev } from '@hubspot/project-parsing-lib';
+import { CLIAccount } from '@hubspot/local-dev-lib/types/Accounts';
+import { getEnv, getConfigAccounts } from '@hubspot/local-dev-lib/config';
+import { getValidEnv } from '@hubspot/local-dev-lib/environment';
+import { ProjectDevArgs } from '../../../types/Yargs';
+import { ProjectConfig } from '../../../types/Projects';
+import { logError } from '../../../lib/errorHandlers';
+import { EXIT_CODES } from '../../../lib/enums/exitCodes';
+import { ensureProjectExists } from '../../../lib/projects';
+import {
+  createInitialBuildForNewProject,
+  createNewProjectForLocalDev,
+  useExistingDevTestAccount,
+  createDeveloperTestAccountForLocalDev,
+} from '../../../lib/localDev';
+import { selectDeveloperTestTargetAccountPrompt } from '../../../lib/prompts/projectDevTargetAccountPrompt';
+import SpinniesManager from '../../../lib/ui/SpinniesManager';
+import LocalDevManagerV2 from '../../../lib/LocalDevManagerV2';
+import { handleExit } from '../../../lib/process';
+import {
+  isAppDeveloperAccount,
+  isStandardAccount,
+} from '../../../lib/accountTypes';
+import { uiCommandReference } from '../../../lib/ui';
+import { i18n } from '../../../lib/lang';
+
+const i18nKey = 'commands.project.subcommands.dev';
+
+export async function unifiedProjectDevFlow(
+  args: ArgumentsCamelCase<ProjectDevArgs>,
+  accountConfig: CLIAccount,
+  projectConfig: ProjectConfig,
+  projectDir: string
+): Promise<void> {
+  logger.log('Unified Apps Local Dev');
+
+  const targetProjectAccountId = args.derivedAccountId;
+  const env = getValidEnv(getEnv(targetProjectAccountId));
+
+  let projectNodes;
+
+  // Get IR
+  try {
+    const intermediateRepresentation = await translateForLocalDev({
+      projectSourceDir: path.join(projectDir, projectConfig.srcDir),
+      platformVersion: projectConfig.platformVersion,
+      accountId: targetProjectAccountId,
+    });
+
+    projectNodes = intermediateRepresentation.intermediateNodesIndexedByUid;
+
+    logger.debug(util.inspect(projectNodes, false, null, true));
+  } catch (e) {
+    if (isTranslationError(e)) {
```

Comment:
not a blocker, but does our standard behavior for `logError` not handle this?

Reply:
Good question, I just pulled this logic from `hs project upload`. Assuming it's something specific to how project-parsing-lib formats errors. Maybe worth moving to `logError` though.

@joe-yeager Is there any reason for manually calling `toString` here?

Reply:
Sorry, missed this. The `toString` method gives a pretty printed message when we want to show more details. 
https://github.com/HubSpot/project-parsing-lib/blob/main/src/lib/errors.ts#L46-L58

We could update the `logError` method to do this if check if we what to centralize it there.